### PR TITLE
hotfix: command and args for job & cronjob

### PR DESCRIPTION
**Pull request by prajjwaldimri**
- Changes command and args to accept input from midgard in different yaml format.

Merge along with frontend and backend PRs

### CODE REVIEW

File: charts/cronjob/templates/cronjob.yaml [hunk 1]
```diff
@@ -185,12 +185,14 @@ spec:
             image: "{{ template "argonaut-deployment.imageName" . }}"
             imagePullPolicy: "Always"
```
Comment (suryaoruganti) [inline charts/cronjob/templates/cronjob.yaml:186]: change this to IfNotPresent
(separate reason)

File: charts/job/templates/job.yaml [hunk 2]
```diff
@@ -179,12 +179,14 @@ spec:
         image: "{{ template "argonaut-deployment.imageName" . }}"
         imagePullPolicy: "Always"
```
Comment (suryaoruganti) [inline charts/job/templates/job.yaml:180]: same as cron

**Review by suryaoruganti — COMMENTED** [review]
minor (different changes).
rest lgtm

File: charts/cronjob/templates/cronjob.yaml (hunk 1 above)
Comment (prajjwaldimri) [reply charts/cronjob/templates/cronjob.yaml]: Changed.

File: charts/job/templates/job.yaml (hunk 2 above)
Comment (prajjwaldimri) [reply charts/job/templates/job.yaml]: Changed

**Comment by suryaoruganti** [comment]
lgtm

**Review by suryaoruganti — APPROVED** [review]

**Pull request merged**